tests/system/providers/amazon/aws/example_emr.py

```diff
@@ -164,10 +164,14 @@ def get_step_id(step_ids: list):
         task_id="add_steps",
         job_flow_id=create_job_flow.output,
         steps=SPARK_STEPS,
-        wait_for_completion=True,
         execution_role_arn=execution_role_arn,
     )
     # [END howto_operator_emr_add_steps]
+    add_steps.wait_for_completion = True
+    # On rare occasion (1 in 50ish?) this system test times out.  Extending
+    # the delay and max_attempts to attempt to mitigate the flaky test.
+    add_steps.waiter_delay = 60
+    add_steps.waiter_max_attempts = 90
```

Comment:
Default values are 30 and 60

Reply:
The new config means we'll wait an hour and half, which feels a bit too long? That would block the whole test suite from completing for that long if this step hung.

If we used to wait 30min total I'd bump it up to 40min or 45min, anything beyond that we really need to find some other fix I think :grimacing:

Reply:
Sure, sounds good.  I'll dial these back a bit.  I feel like maybe adding more retries at the default duration then?  That would mean we can still succeed sooner, but get to try longer.